Project/Assets/Scripts/Player Code/MotionModes/AnimatedRunMode.cs

```diff
@@ -122,9 +128,8 @@ public override void ApplyMotion(CharacterViewModel characterViewModel)
 			var playerSpeed = characterRigidbody.velocity.x;
 			if (SpeedInRange(playerSpeed))
 			{
-				//characterRigidbody.AddForce(currentSpeed * Vector2.right, ForceMode);
-                characterRigidbody.velocity = new Vector3(currentSpeed * 5, 0, 0);
-            }
+               			UpdateHorizontalVelocity(SpeedMultiplier);
```

Comment:
W tym miejscu powinno być `playerSpeed * SpeedMutliplier`

Reply:
A racja, głupie niedopatrzenie, dzięki. Natomiast dałem currentSpeed zamiast playerSpeed. playerSpeed to characterRigidbody.velocity.x, a przy początku ruchu jest = 0, więc nie ruszyłby się.